SU-HSDO/suhumsci
# Hsd8 426 capx

# READY FOR REVIEW

# Summary
- CAPx Importer

# Needed By (Date)
- 10/24

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. `composer update --prefer-source`
1. `blt drupal:sync --site=archaeology`
1. navigate to `/admin/structure/migrate/manage/hs_capx/migrations`
1. ensure no errors and there are 2 importers
1. navigate to `/admin/structure/migrate/capx/creds` input credentials (You probably need a new key)
1. navigate to `/admin/structure/migrate/capx` and choose a workgroup and/or organization and save
1. run cron
1. verify new content has been created and some fields are populated (if data is available) like:
    * first name
    * last name
    * bio
    * profile Picture
    * email address
    * Telephone number
    * Title

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
